armory/datasets/generator.py

```diff
@@ -164,3 +167,13 @@ def wrap_generator(armory_data_generator):
     from armory.datasets import art_wrapper
 
     return art_wrapper.WrappedDataGenerator(armory_data_generator)
+
+
+class EvalGenerator:
+    """
+    Wraps a specified number of batches in a DataGenerator to allow for evaluating on
+    part of a dataset when running through a scenario
+    """
+
+    def __init__(self, armory_generator, num_eval_batches):
+        raise NotImplementedError("EvalGenerator not implemented")
```

Comment:
Added to satisfy import statement in another file I had not changed; should this be implemented or removed? I can find where the specific site where this needed to be imported if it is helpful.